main.c

```diff
@@ -708,7 +738,7 @@ int main(int argc, char *argv[]) {
 
 	int opt;
 	char *format = "%x,%y %wx%h";
```

Comment:
If the `-o` option is used, should a different default format be used that includes the output name (or maybe is only the output name)?